Dockerfile

```diff
@@ -12,6 +12,9 @@ RUN unzip protoc-3.11.4-linux-x86_64.zip -d protoc3 && \
 RUN go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
 RUN go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
 RUN go get -u github.com/golang/protobuf/protoc-gen-go
+RUN go get -u github.com/gogo/protobuf/proto
```

Comment:
Please update the readme stating why this is needed or add an example that makes it clear